File: Python/model/otp.py
from pydantic import BaseModel

class OtpRequest(BaseModel):
    phone: str
    country_code: str = 91
    otp_length: int = 6
    flow_type: str = "SMS"

class VerifyOtp(BaseModel):
    phone: str
    otp: str
    verification_id: str

File: Go/model/otp.go
package model

type OTPRequest struct {
	Phone string `json:"phone"`
	Country int `json:"country" default:"91"`
	OTPLength int `json:"otp_length" default:"6"`
	FlowType string `json:"flow_type" default:"SMS"`
}

type OTPResponse struct {
	Phone string `json:"phone"`
	OTP string `json:"otp"`
	VerificationID string `json:"verification_id"`
}

type VerifyOTPRequest struct {
	Phone string `json:"phone"`
	OTP string `json:"otp"`
	VerificationID string `json:"verification_id"`
	AuthToken string `json:"auth_token"`
}